src/components/TodoList/TodoList.tsx

```diff
@@ -1,224 +1,89 @@
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">10</td>
-
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">
-                illo est ratione doloremque quia maiores aut
-              </p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
+  return (
+    <>
+      {!filteredTodos.length && (
+        <tr>
+          <td>
+            <p className="notification is-warning">
+              There are no todos matching current filter criteria
+            </p>
+          </td>
+        </tr>
+      )}
+
+      {filteredTodos.length > 0 && (
+        <table className="table is-narrow is-fullwidth">
+          <thead>
+            <tr>
+              <th>#</th>
+              <th>
                 <span className="icon">
-                  <i className="far fa-eye" />
+                  <i className="fas fa-check" />
                 </span>
-              </button>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+              </th>
+              <th>Title</th>
+              <th> </th>
+            </tr>
+          </thead>
+          <tbody>
+            {filteredTodos.map(todo => (
+              <tr data-cy="todo" key={todo.id}>
+                <td className="is-vcentered">{todo.id}</td>
+                <td className={cn('is-vcentered', { 'has-text-success': todo.completed, 'has-text-danger': !todo.completed })}>
+                  {todo.completed && (
+                    <span className="icon" data-cy="iconCompleted">
+                      <i className="fas fa-check" />
+                    </span>
+                  )}
+                </td>
+                <td className="is-vcentered is-expanded">
+                  <p className={cn({ 'has-text-success': todo.completed, 'has-text-danger': !todo.completed })}>{todo.title}</p>
+                </td>
+                <td className="has-text-right is-vcentered">
+                  <button
+                    data-cy="selectButton"
+                    className="button"
+                    type="button"
+                    onClick={() => setCurrentTodo(todo)}
+                  >
+                    <span className="icon">
+                      <i
+                        className={
+                          todo.id === currentTodo?.id
+                            ? 'far fa-eye-slash'
+                            : 'far fa-eye'
+                        }
+                        key={todo.id}
+                      />
+                    </span>
+                  </button>
+                </td>
+              </tr>
```

Comment:
Consider creating `<Todo />` component and use destruction assignment to `todo` object